### PR TITLE
add word backup to everywhere except dashboard page

### LINKED ISSUE #828: Add "backups" text to link describing last backup taken

### PR DESCRIPTION
**Pull request by abemassry**
closes #828 

Linodes page:
List view:
image
Card view:
image

Dashboard page:
image

### CODE REVIEW

File: src/linodes/components/Linode.js [hunk 1]
```diff
@@ -89,35 +89,42 @@ export function getNextBackup(linode) {
...
+          {backupWord}{up('i')}n ~{nextBackup.fromNow(true)}
         </Link>);
     }
 
     const backupStatus = linode.backups.last_backup.status;
     if (backupStatus === 'running') {
       return (
         <Link className="backup-status" to={`/linodes/${linode.label}/backups`}>
-          Running
+          {backupWord}{up('r')}unning
```
Comment (eatonphil) [inline src/linodes/components/Linode.js]: If a backup is currently running, that information is displayed on the dashboard page?

File: src/linodes/components/Linode.js [hunk 2]
```diff
@@ -89,35 +89,42 @@ export function getNextBackup(linode) {
...
         </Link>);
     }
 
     if (backupStatus === 'pending') {
       return (
         <Link className="backup-status" to={`/linodes/${linode.label}/backups`}>
-          Pending
+          {backupWord}{up('p')}ending
         </Link>);
```
Comment (eatonphil) [inline src/linodes/components/Linode.js]: @briansteffens looking through all this code, looks like backups might be another good candidate for a new src/api/objects/backups file with this formatting taken care of. Idk what your impression of the src/api/objects/event file has been though.

Comment (briansteffens) [inline src/linodes/components/Linode.js]: Yeah we definitely need something like that.

The src/api/objects/event file seems to work well, my only hesitation with it is it's not super obvious that it's a kind of convenience wrapper object. Maybe naming it something else would help make its use clear?

I think the easiest/most obvious thing would be to have functions located somewhere that you import as needed.

Comment (eatonphil) [inline src/linodes/components/Linode.js]: Agreed that the naming is not good. Totally welcome to any suggestions for it: src/api/wrappers/Event.js? idk

Comment (briansteffens) [inline src/linodes/components/Linode.js]: Yeah that looks better.

Comment (eatonphil) [inline src/linodes/components/Linode.js]: https://github.com/linode/manager/issues/967

**Pull request merged**